src/ripple/basics/strHex.h

```diff
@@ -94,6 +94,14 @@ inline std::string strHex (const std::uint64_t uiHost)
     auto const end   = begin + sizeof(uBig);
     return strHex(begin, end);
 }
+
+//converts data to a string of bytes
+template <class T>
+std::string toBytes(T const & data)
```

Comment:
We can definitely just use those overloads; that will be more efficient anyways since we don't have to create a `std::string` temporary. Will remove this `toBytes()` function

Reply:
removed this function and used the other overloads